lib/razorpay.js

```diff
@@ -25,7 +25,7 @@ class Razorpay {
     this.key_secret = key_secret
 
     this.api = new API({
-      hostUrl: 'https://api.razorpay.com/v1/',
+      hostUrl: 'https://api.razorpay.com',
```

Comment:
API version is now passed as params? Can you attach any slack thread where this has been discussed and finalised?

Reply:
@1995navinkumar we have to integrate v2 [Partner api's](https://razorpay.com/docs/api/partners/account-onboarding/#create-an-account) and this same approach we used in [php](https://github.com/razorpay/razorpay-php/pull/348/files#diff-edd60a8e87a801b7bf604385eadf3f002ee7a58fa86d53b2b3e5baacc4b94e82R7) as well as [Java](https://github.com/razorpay/razorpay-java/pull/286/files#diff-be96186a4d128a00825e9baefded1873f4a560c1df1b877262e598f4b4828184R71) 

Thread : https://razorpay.slack.com/archives/C04DAUDQW9G/p1683794805871459